src/nyc_trees/apps/survey/templates/survey/reserve_blockface.html

```diff
@@ -38,6 +38,107 @@
 
 {% endblock main %}
 
+{% block extra_content %}
+<div class="modal fade" id="limit-reached-modal">
```

Comment:
I probably would have solved this problem with one modal and text mutations, but this is a little more elegant.